### PR TITLE
New color in hover to add flat

### PR DESCRIPTION
**Pull request by adam-krukowski**
I've changed color from grey to light blue

### CODE REVIEW

File: app/assets/stylesheets/website.css [hunk 1]
```diff
@@ -23,8 +23,8 @@
 
 /* Change color on hover */
 .navbar a:hover {
-  background-color: #cfdae6; /* Medium gray background on hover */
-  color: #ffffff; /* White text on hover */
+  background-color: #052d57; /* Medium gray background on hover */
+  color: #92b8ed; /* White text on hover */
```
Comment (Patrycja-Nowak) [inline app/assets/stylesheets/website.css]: Can you please update the comments to match the colours? The background is now navy and the text colour is light blue

**Review by Patrycja-Nowak — CHANGES_REQUESTED** [review]
Can you update your comments to match the colours you changed to?

File: app/assets/stylesheets/website.css (hunk 1 above)
Comment (adam-krukowski) [reply app/assets/stylesheets/website.css]: Hi thank you for review. I changed it

**Review by Patrycja-Nowak — APPROVED** [review]
LGTM 👍

**Pull request merged**